paimon-spark/paimon-spark-4.0/pom.xml

```diff
@@ -28,8 +28,8 @@ under the License.
         <version>1.3-SNAPSHOT</version>
     </parent>
 
-    <artifactId>paimon-spark-4.0</artifactId>
-    <name>Paimon : Spark : 4.0</name>
+    <artifactId>paimon-spark-4.0_${scala.binary.version}</artifactId>
```

Comment:
Spark4.0 has dropped scala2.12. Unless Spark4.x support other higher version, we don't allow to build spark with other scala version.

Reply:
Thank you for the review. The reason I added this is to see if Spark, when producing the Spark 4.0 package, also includes the Scala version, i.e., `spark-core_2.13`. Do you think it's necessary to include the Scala version such as `paimon-spark-4.0_2.13`, or should it just be deployed as `paimon-spark-4.0`?

Reply:
It doesn't matter whether the module name incudes scala version. But I think we should not allow to use scala 2.12 to build paimon-spark-4.x.

Reply:
Got it, thanks.